### PR TITLE
WEBSDK-13 stop throwing error if init fails

### PR DESCRIPTION
**Pull request by yntema**

### CODE REVIEW

File: src/6_branch.js [hunk 1]
```diff
@@ -66,9 +66,7 @@ var wrap = function(parameters, func, init) {
 		var lastArg = arguments[arguments.length - 1];
 		if (parameters === callback_params.NO_CALLBACK || typeof lastArg !== 'function') {
 			callback = function(err) {
-				if (err) {
-					throw err;
-				}
```
Comment (zeeshabnc) [inline src/6_branch.js]: Can we add a console.log statement here?

Comment (yntema) [inline src/6_branch.js]: We can, but it seemed repetitive since we already log the error else where (line 103).

Comment (zeeshabnc) [inline src/6_branch.js]: ok

**Comment by jsaleigh** [comment]
👍

**Pull request merged**